TVOS360Video.podspec

```diff
@@ -9,10 +9,10 @@ Pod::Spec.new do |s|
     It provides no control user interface; it is intended to be embedded in your own video player implementation.
     DESC
 
-    s.homepage         = 'https://github.com/nytm/ios-360-videos/'
+    s.homepage         = 'https://github.com/socialvibe/ios-360-videos'
```

Comment:
i am curious why these were not part of the previous changes for 1.1.2, etc.

Reply:
Yeah, it looks like 5 years we just sort of slapped together some changes and rushed a release out without bothering to remove the old podspec, change these URLs, or merge anything into `master` or `develop`.